extras/pahma/app/views/catalog/_document_slideshow.html.erb

```diff
@@ -0,0 +1,33 @@
+<% view_config = local_assigns[:view_config] || blacklight_config&.view_config(document_index_view_type) %>
```

Comment:
Overwriting this file to add aria attributes (formerly done via blacklight_gallery.js) and make the #documents element focusable.